@here/harp-mapview/lib/Tile.ts

```diff
@@ -995,6 +996,18 @@ export class Tile implements CachedResource {
         return this.projection.worldExtent(0, 0).max.x * this.offset;
     }
 
+    /**
+     * Update tile for current map view zoom level
+     * @param zoomLevel Zoom level of the map view
+     */
+    update(zoomLevel: number): void {
+        for (const object of this.objects) {
+            if (object instanceof LodMesh) {
+                object.setLevelOfDetail(zoomLevel - this.tileKey.level);
```

Comment:
In which cases would `zoomLevel != this.tileKey.level`. During fallback phase `zoomLevel` can also be smaller than `this.tileKey.level`

Reply:
When mixed LOD is enabled the tile key level can be smaller than the display zoom level, e.g., on the poles or high camera tilt. This PR basically fixes the cracks so that tiles of different zoom levels will use the same level of tessellation -> the edges of tile geometry will fit.
I can't simply increase tessellation of lower zoom levels because this will only change at which zoom level the cracks are visible and not remove them completly.